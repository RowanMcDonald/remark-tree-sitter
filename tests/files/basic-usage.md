# Hello World

`````some-language
This is a code snippet
`````

```
This is a code snippet
```

```typescript
function foo() {
  return 1;
}
```

```ts
function foo() {
  return 1;
}
```
