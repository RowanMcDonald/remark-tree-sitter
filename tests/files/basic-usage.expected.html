<h1>Hello World</h1>
<pre><code class="language-some-language">This is a code snippet
</code></pre>
<pre><code>This is a code snippet
</code></pre>
<pre><code class="tree-sitter language-typescript"><span class="source ts"><span class="storage type function">function</span> <span class="entity name function">foo</span><span class="punctuation definition parameters begin bracket round">(</span><span class="punctuation definition parameters end bracket round">)</span> <span class="punctuation definition function body begin bracket curly">{</span>
  <span class="keyword control">return</span> <span class="constant numeric">1</span><span class="punctuation terminator statement semicolon">;</span>
<span class="punctuation definition function body end bracket curly">}</span></span></code></pre>
<pre><code class="language-ts">function foo() {
  return 1;
}
</code></pre>